commit a93fecdcece3e2178834f4b4868b2309b0158753
Author: Mike FABIAN <mfabian@redhat.com>
Date:   Wed Jan 31 06:18:47 2018 +0100

    iso14651_t1_common: make the fourth level the codepoint for characters which are ignorable on all 4 levels
    
    Entries for characters which have “IGNORE” on all 4 levels like:
    
     <U0001> IGNORE;IGNORE;IGNORE;IGNORE % START OF HEADING (in ISO 6429)
    
    are changed into:
    
     <U0001> IGNORE;IGNORE;IGNORE;<U0001> % START OF HEADING (in ISO 6429)
    
    i.e. putting the code point of the character into the fourth level
    instead of “IGNORE”. Without that change, all such characters
    would compare equal which would make a wcscoll test case fail.
    It is better to have a clearly defined sort order even for characters
    like this so it is good to use the code point as a tie-break.
    
            * localedata/locales/iso14651_t1_common: Use the code point of a
            character in the fourth collation level instead of IGNORE for all
            entries which have IGNORE on all 4 levels.

diff --git a/localedata/locales/iso14651_t1_common b/localedata/locales/iso14651_t1_common
index 443e689da0ee03dc..2d5fdfa87bb93fc8 100644
--- a/localedata/locales/iso14651_t1_common
+++ b/localedata/locales/iso14651_t1_common
@@ -54817,463 +54817,463 @@ endif
 
 % <Uxxxx> <Base>;<Accent>;<Case>;<Special>
 
-<U0000> IGNORE;IGNORE;IGNORE;IGNORE % NULL (in ISO 6429)
-<U0001> IGNORE;IGNORE;IGNORE;IGNORE % START OF HEADING (in ISO 6429)
-<U0002> IGNORE;IGNORE;IGNORE;IGNORE % START OF TEXT (in ISO 6429)
-<U0003> IGNORE;IGNORE;IGNORE;IGNORE % END OF TEXT (in ISO 6429)
-<U0004> IGNORE;IGNORE;IGNORE;IGNORE % END OF TRANSMISSION (in ISO 6429)
-<U0005> IGNORE;IGNORE;IGNORE;IGNORE % ENQUIRY (in ISO 6429)
-<U0006> IGNORE;IGNORE;IGNORE;IGNORE % ACKNOWLEDGE (in ISO 6429)
-<U0007> IGNORE;IGNORE;IGNORE;IGNORE % BELL (in ISO 6429)
-<U0008> IGNORE;IGNORE;IGNORE;IGNORE % BACKSPACE (in ISO 6429)
-<U000E> IGNORE;IGNORE;IGNORE;IGNORE % SHIFT OUT (in ISO 6429)
-<U000F> IGNORE;IGNORE;IGNORE;IGNORE % SHIFT IN (in ISO 6429)
-<U0010> IGNORE;IGNORE;IGNORE;IGNORE % DATA LINK ESCAPE (in ISO 6429)
-<U0011> IGNORE;IGNORE;IGNORE;IGNORE % DEVICE CONTROL ONE (in ISO 6429)
-<U0012> IGNORE;IGNORE;IGNORE;IGNORE % DEVICE CONTROL TWO (in ISO 6429)
-<U0013> IGNORE;IGNORE;IGNORE;IGNORE % DEVICE CONTROL THREE (in ISO 6429)
-<U0014> IGNORE;IGNORE;IGNORE;IGNORE % DEVICE CONTROL FOUR (in ISO 6429)
-<U0015> IGNORE;IGNORE;IGNORE;IGNORE % NEGATIVE ACKNOWLEDGE (in ISO 6429)
-<U0016> IGNORE;IGNORE;IGNORE;IGNORE % SYNCHRONOUS IDLE (in ISO 6429)
-<U0017> IGNORE;IGNORE;IGNORE;IGNORE % END OF TRANSMISSION BLOCK (in ISO 6429)
-<U0018> IGNORE;IGNORE;IGNORE;IGNORE % CANCEL (in ISO 6429)
-<U0019> IGNORE;IGNORE;IGNORE;IGNORE % END OF MEDIUM (in ISO 6429)
-<U001A> IGNORE;IGNORE;IGNORE;IGNORE % SUBSTITUTE (in ISO 6429)
-<U001B> IGNORE;IGNORE;IGNORE;IGNORE % ESCAPE (in ISO 6429)
-<U001C> IGNORE;IGNORE;IGNORE;IGNORE % FILE SEPARATOR (in ISO 6429)
-<U001D> IGNORE;IGNORE;IGNORE;IGNORE % GROUP SEPARATOR (in ISO 6429)
-<U001E> IGNORE;IGNORE;IGNORE;IGNORE % RECORD SEPARATOR (in ISO 6429)
-<U001F> IGNORE;IGNORE;IGNORE;IGNORE % UNIT SEPARATOR (in ISO 6429)
-<U007F> IGNORE;IGNORE;IGNORE;IGNORE % DELETE (in ISO 6429)
-<U0080> IGNORE;IGNORE;IGNORE;IGNORE % <control>
-<U0081> IGNORE;IGNORE;IGNORE;IGNORE % <control>
-<U0082> IGNORE;IGNORE;IGNORE;IGNORE % BREAK PERMITTED HERE (in ISO 6429)
-<U0083> IGNORE;IGNORE;IGNORE;IGNORE % NO BREAK HERE (in ISO 6429)
-<U0084> IGNORE;IGNORE;IGNORE;IGNORE % <control>
-<U0086> IGNORE;IGNORE;IGNORE;IGNORE % START OF SELECTED AREA (in ISO 6429)
-<U0087> IGNORE;IGNORE;IGNORE;IGNORE % END OF SELECTED AREA (in ISO 6429)
-<U0088> IGNORE;IGNORE;IGNORE;IGNORE % CHARACTER TABULATION SET (in ISO 6429)
-<U0089> IGNORE;IGNORE;IGNORE;IGNORE % CHARACTER TABULATION WITH JUSTIFICATION (in ISO 6429)
-<U008A> IGNORE;IGNORE;IGNORE;IGNORE % LINE TABULATION SET (in ISO 6429)
-<U008B> IGNORE;IGNORE;IGNORE;IGNORE % PARTIAL LINE FORWARD (in ISO 6429)
-<U008C> IGNORE;IGNORE;IGNORE;IGNORE % PARTIAL LINE BACKWARD (in ISO 6429)
-<U008D> IGNORE;IGNORE;IGNORE;IGNORE % PARTIAL LINE FEED (in ISO 6429)
-<U008E> IGNORE;IGNORE;IGNORE;IGNORE % SINGLE SHIFT TWO (in ISO 6429)
-<U008F> IGNORE;IGNORE;IGNORE;IGNORE % SINGLE SHIFT THREE (in ISO 6429)
-<U0090> IGNORE;IGNORE;IGNORE;IGNORE % DEVICE CONTROL STRING (in ISO 6429)
-<U0091> IGNORE;IGNORE;IGNORE;IGNORE % PRIVATE USE ONE (in ISO 6429)
-<U0092> IGNORE;IGNORE;IGNORE;IGNORE % PRIVATE USE TWO (in ISO 6429)
-<U0093> IGNORE;IGNORE;IGNORE;IGNORE % SET TRANSMIT STATE (in ISO 6429)
-<U0094> IGNORE;IGNORE;IGNORE;IGNORE % CANCEL CHARACTER (in ISO 6429)
-<U0095> IGNORE;IGNORE;IGNORE;IGNORE % MESSAGE WAITING (in ISO 6429)
-<U0096> IGNORE;IGNORE;IGNORE;IGNORE % START OF GUARDED AREA (in ISO 6429)
-<U0097> IGNORE;IGNORE;IGNORE;IGNORE % END OF GUARDED AREA (in ISO 6429)
-<U0098> IGNORE;IGNORE;IGNORE;IGNORE % START OF STRING (in ISO 6429)
-<U0099> IGNORE;IGNORE;IGNORE;IGNORE % <control>
-<U009A> IGNORE;IGNORE;IGNORE;IGNORE % SINGLE CHARACTER INTRODUCER (in ISO 6429)
-<U009B> IGNORE;IGNORE;IGNORE;IGNORE % CONTROL SEQUENCE INTRODUCER (in ISO 6429)
-<U009C> IGNORE;IGNORE;IGNORE;IGNORE % STRING TERMINATOR (in ISO 6429)
-<U009D> IGNORE;IGNORE;IGNORE;IGNORE % OPERATING SYSTEM COMMAND (in ISO 6429)
-<U009E> IGNORE;IGNORE;IGNORE;IGNORE % PRIVACY MESSAGE (in ISO 6429)
-<U009F> IGNORE;IGNORE;IGNORE;IGNORE % APPLICATION PROGRAM COMMAND (in ISO 6429)
-<U00AD> IGNORE;IGNORE;IGNORE;IGNORE % SOFT HYPHEN
-<U061C> IGNORE;IGNORE;IGNORE;IGNORE % ARABIC LETTER MARK
-<U070F> IGNORE;IGNORE;IGNORE;IGNORE % SYRIAC ABBREVIATION MARK
-<U08E2> IGNORE;IGNORE;IGNORE;IGNORE % ARABIC DISPUTED END OF AYAH
-<U180B> IGNORE;IGNORE;IGNORE;IGNORE % MONGOLIAN FREE VARIATION SELECTOR ONE
-<U180C> IGNORE;IGNORE;IGNORE;IGNORE % MONGOLIAN FREE VARIATION SELECTOR TWO
-<U180D> IGNORE;IGNORE;IGNORE;IGNORE % MONGOLIAN FREE VARIATION SELECTOR THREE
-<U180E> IGNORE;IGNORE;IGNORE;IGNORE % MONGOLIAN VOWEL SEPARATOR
-<U200B> IGNORE;IGNORE;IGNORE;IGNORE % ZERO WIDTH SPACE
-<U200C> IGNORE;IGNORE;IGNORE;IGNORE % ZERO WIDTH NON-JOINER
-<U200D> IGNORE;IGNORE;IGNORE;IGNORE % ZERO WIDTH JOINER
-<U200E> IGNORE;IGNORE;IGNORE;IGNORE % LEFT-TO-RIGHT MARK
-<U200F> IGNORE;IGNORE;IGNORE;IGNORE % RIGHT-TO-LEFT MARK
-<U202A> IGNORE;IGNORE;IGNORE;IGNORE % LEFT-TO-RIGHT EMBEDDING
-<U202B> IGNORE;IGNORE;IGNORE;IGNORE % RIGHT-TO-LEFT EMBEDDING
-<U202C> IGNORE;IGNORE;IGNORE;IGNORE % POP DIRECTIONAL FORMATTING
-<U202D> IGNORE;IGNORE;IGNORE;IGNORE % LEFT-TO-RIGHT OVERRIDE
-<U202E> IGNORE;IGNORE;IGNORE;IGNORE % RIGHT-TO-LEFT OVERRIDE
-<U2060> IGNORE;IGNORE;IGNORE;IGNORE % WORD JOINER
-<U2066> IGNORE;IGNORE;IGNORE;IGNORE % LEFT-TO-RIGHT ISOLATE
-<U2067> IGNORE;IGNORE;IGNORE;IGNORE % RIGHT-TO-LEFT ISOLATE
-<U2068> IGNORE;IGNORE;IGNORE;IGNORE % FIRST STRONG ISOLATE
-<U2069> IGNORE;IGNORE;IGNORE;IGNORE % POP DIRECTIONAL ISOLATE
-<U206A> IGNORE;IGNORE;IGNORE;IGNORE % INHIBIT SYMMETRIC SWAPPING
-<U206B> IGNORE;IGNORE;IGNORE;IGNORE % ACTIVATE SYMMETRIC SWAPPING
-<U206C> IGNORE;IGNORE;IGNORE;IGNORE % INHIBIT ARABIC FORM SHAPING
-<U206D> IGNORE;IGNORE;IGNORE;IGNORE % ACTIVATE ARABIC FORM SHAPING
-<U206E> IGNORE;IGNORE;IGNORE;IGNORE % NATIONAL DIGIT SHAPES
-<U206F> IGNORE;IGNORE;IGNORE;IGNORE % NOMINAL DIGIT SHAPES
-<UFE00> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-1
-<UFE01> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-2
-<UFE02> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-3
-<UFE03> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-4
-<UFE04> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-5
-<UFE05> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-6
-<UFE06> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-7
-<UFE07> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-8
-<UFE08> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-9
-<UFE09> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-10
-<UFE0A> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-11
-<UFE0B> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-12
-<UFE0C> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-13
-<UFE0D> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-14
-<UFE0E> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-15
-<UFE0F> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-16
-<UFEFF> IGNORE;IGNORE;IGNORE;IGNORE % ZERO WIDTH NO-BREAK SPACE
-<UFFF9> IGNORE;IGNORE;IGNORE;IGNORE % INTERLINEAR ANNOTATION ANCHOR
-<UFFFA> IGNORE;IGNORE;IGNORE;IGNORE % INTERLINEAR ANNOTATION SEPARATOR
-<UFFFB> IGNORE;IGNORE;IGNORE;IGNORE % INTERLINEAR ANNOTATION TERMINATOR
-<U0001BCA0> IGNORE;IGNORE;IGNORE;IGNORE % SHORTHAND FORMAT LETTER OVERLAP
-<U0001BCA1> IGNORE;IGNORE;IGNORE;IGNORE % SHORTHAND FORMAT CONTINUING OVERLAP
-<U0001BCA2> IGNORE;IGNORE;IGNORE;IGNORE % SHORTHAND FORMAT DOWN STEP
-<U0001BCA3> IGNORE;IGNORE;IGNORE;IGNORE % SHORTHAND FORMAT UP STEP
-<U0001D173> IGNORE;IGNORE;IGNORE;IGNORE % MUSICAL SYMBOL BEGIN BEAM
-<U0001D174> IGNORE;IGNORE;IGNORE;IGNORE % MUSICAL SYMBOL END BEAM
-<U0001D175> IGNORE;IGNORE;IGNORE;IGNORE % MUSICAL SYMBOL BEGIN TIE
-<U0001D176> IGNORE;IGNORE;IGNORE;IGNORE % MUSICAL SYMBOL END TIE
-<U0001D177> IGNORE;IGNORE;IGNORE;IGNORE % MUSICAL SYMBOL BEGIN SLUR
-<U0001D178> IGNORE;IGNORE;IGNORE;IGNORE % MUSICAL SYMBOL END SLUR
-<U0001D179> IGNORE;IGNORE;IGNORE;IGNORE % MUSICAL SYMBOL BEGIN PHRASE
-<U0001D17A> IGNORE;IGNORE;IGNORE;IGNORE % MUSICAL SYMBOL END PHRASE
-<U000E0001> IGNORE;IGNORE;IGNORE;IGNORE % LANGUAGE TAG
-<U000E0020> IGNORE;IGNORE;IGNORE;IGNORE % TAG SPACE
-<U000E0021> IGNORE;IGNORE;IGNORE;IGNORE % TAG EXCLAMATION MARK
-<U000E0022> IGNORE;IGNORE;IGNORE;IGNORE % TAG QUOTATION MARK
-<U000E0023> IGNORE;IGNORE;IGNORE;IGNORE % TAG NUMBER SIGN
-<U000E0024> IGNORE;IGNORE;IGNORE;IGNORE % TAG DOLLAR SIGN
-<U000E0025> IGNORE;IGNORE;IGNORE;IGNORE % TAG PERCENT SIGN
-<U000E0026> IGNORE;IGNORE;IGNORE;IGNORE % TAG AMPERSAND
-<U000E0027> IGNORE;IGNORE;IGNORE;IGNORE % TAG APOSTROPHE
-<U000E0028> IGNORE;IGNORE;IGNORE;IGNORE % TAG LEFT PARENTHESIS
-<U000E0029> IGNORE;IGNORE;IGNORE;IGNORE % TAG RIGHT PARENTHESIS
-<U000E002A> IGNORE;IGNORE;IGNORE;IGNORE % TAG ASTERISK
-<U000E002B> IGNORE;IGNORE;IGNORE;IGNORE % TAG PLUS SIGN
-<U000E002C> IGNORE;IGNORE;IGNORE;IGNORE % TAG COMMA
-<U000E002D> IGNORE;IGNORE;IGNORE;IGNORE % TAG HYPHEN-MINUS
-<U000E002E> IGNORE;IGNORE;IGNORE;IGNORE % TAG FULL STOP
-<U000E002F> IGNORE;IGNORE;IGNORE;IGNORE % TAG SOLIDUS
-<U000E0030> IGNORE;IGNORE;IGNORE;IGNORE % TAG DIGIT ZERO
-<U000E0031> IGNORE;IGNORE;IGNORE;IGNORE % TAG DIGIT ONE
-<U000E0032> IGNORE;IGNORE;IGNORE;IGNORE % TAG DIGIT TWO
-<U000E0033> IGNORE;IGNORE;IGNORE;IGNORE % TAG DIGIT THREE
-<U000E0034> IGNORE;IGNORE;IGNORE;IGNORE % TAG DIGIT FOUR
-<U000E0035> IGNORE;IGNORE;IGNORE;IGNORE % TAG DIGIT FIVE
-<U000E0036> IGNORE;IGNORE;IGNORE;IGNORE % TAG DIGIT SIX
-<U000E0037> IGNORE;IGNORE;IGNORE;IGNORE % TAG DIGIT SEVEN
-<U000E0038> IGNORE;IGNORE;IGNORE;IGNORE % TAG DIGIT EIGHT
-<U000E0039> IGNORE;IGNORE;IGNORE;IGNORE % TAG DIGIT NINE
-<U000E003A> IGNORE;IGNORE;IGNORE;IGNORE % TAG COLON
-<U000E003B> IGNORE;IGNORE;IGNORE;IGNORE % TAG SEMICOLON
-<U000E003C> IGNORE;IGNORE;IGNORE;IGNORE % TAG LESS-THAN SIGN
-<U000E003D> IGNORE;IGNORE;IGNORE;IGNORE % TAG EQUALS SIGN
-<U000E003E> IGNORE;IGNORE;IGNORE;IGNORE % TAG GREATER-THAN SIGN
-<U000E003F> IGNORE;IGNORE;IGNORE;IGNORE % TAG QUESTION MARK
-<U000E0040> IGNORE;IGNORE;IGNORE;IGNORE % TAG COMMERCIAL AT
-<U000E0041> IGNORE;IGNORE;IGNORE;IGNORE % TAG LATIN CAPITAL LETTER A
-<U000E0042> IGNORE;IGNORE;IGNORE;IGNORE % TAG LATIN CAPITAL LETTER B
-<U000E0043> IGNORE;IGNORE;IGNORE;IGNORE % TAG LATIN CAPITAL LETTER C
-<U000E0044> IGNORE;IGNORE;IGNORE;IGNORE % TAG LATIN CAPITAL LETTER D
-<U000E0045> IGNORE;IGNORE;IGNORE;IGNORE % TAG LATIN CAPITAL LETTER E
-<U000E0046> IGNORE;IGNORE;IGNORE;IGNORE % TAG LATIN CAPITAL LETTER F
-<U000E0047> IGNORE;IGNORE;IGNORE;IGNORE % TAG LATIN CAPITAL LETTER G
-<U000E0048> IGNORE;IGNORE;IGNORE;IGNORE % TAG LATIN CAPITAL LETTER H
-<U000E0049> IGNORE;IGNORE;IGNORE;IGNORE % TAG LATIN CAPITAL LETTER I
-<U000E004A> IGNORE;IGNORE;IGNORE;IGNORE % TAG LATIN CAPITAL LETTER J
-<U000E004B> IGNORE;IGNORE;IGNORE;IGNORE % TAG LATIN CAPITAL LETTER K
-<U000E004C> IGNORE;IGNORE;IGNORE;IGNORE % TAG LATIN CAPITAL LETTER L
-<U000E004D> IGNORE;IGNORE;IGNORE;IGNORE % TAG LATIN CAPITAL LETTER M
-<U000E004E> IGNORE;IGNORE;IGNORE;IGNORE % TAG LATIN CAPITAL LETTER N
-<U000E004F> IGNORE;IGNORE;IGNORE;IGNORE % TAG LATIN CAPITAL LETTER O
-<U000E0050> IGNORE;IGNORE;IGNORE;IGNORE % TAG LATIN CAPITAL LETTER P
-<U000E0051> IGNORE;IGNORE;IGNORE;IGNORE % TAG LATIN CAPITAL LETTER Q
-<U000E0052> IGNORE;IGNORE;IGNORE;IGNORE % TAG LATIN CAPITAL LETTER R
-<U000E0053> IGNORE;IGNORE;IGNORE;IGNORE % TAG LATIN CAPITAL LETTER S
-<U000E0054> IGNORE;IGNORE;IGNORE;IGNORE % TAG LATIN CAPITAL LETTER T
-<U000E0055> IGNORE;IGNORE;IGNORE;IGNORE % TAG LATIN CAPITAL LETTER U
-<U000E0056> IGNORE;IGNORE;IGNORE;IGNORE % TAG LATIN CAPITAL LETTER V
-<U000E0057> IGNORE;IGNORE;IGNORE;IGNORE % TAG LATIN CAPITAL LETTER W
-<U000E0058> IGNORE;IGNORE;IGNORE;IGNORE % TAG LATIN CAPITAL LETTER X
-<U000E0059> IGNORE;IGNORE;IGNORE;IGNORE % TAG LATIN CAPITAL LETTER Y
-<U000E005A> IGNORE;IGNORE;IGNORE;IGNORE % TAG LATIN CAPITAL LETTER Z
-<U000E005B> IGNORE;IGNORE;IGNORE;IGNORE % TAG LEFT SQUARE BRACKET
-<U000E005C> IGNORE;IGNORE;IGNORE;IGNORE % TAG REVERSE SOLIDUS
-<U000E005D> IGNORE;IGNORE;IGNORE;IGNORE % TAG RIGHT SQUARE BRACKET
-<U000E005E> IGNORE;IGNORE;IGNORE;IGNORE % TAG CIRCUMFLEX ACCENT
-<U000E005F> IGNORE;IGNORE;IGNORE;IGNORE % TAG LOW LINE
-<U000E0060> IGNORE;IGNORE;IGNORE;IGNORE % TAG GRAVE ACCENT
-<U000E0061> IGNORE;IGNORE;IGNORE;IGNORE % TAG LATIN SMALL LETTER A
-<U000E0062> IGNORE;IGNORE;IGNORE;IGNORE % TAG LATIN SMALL LETTER B
-<U000E0063> IGNORE;IGNORE;IGNORE;IGNORE % TAG LATIN SMALL LETTER C
-<U000E0064> IGNORE;IGNORE;IGNORE;IGNORE % TAG LATIN SMALL LETTER D
-<U000E0065> IGNORE;IGNORE;IGNORE;IGNORE % TAG LATIN SMALL LETTER E
-<U000E0066> IGNORE;IGNORE;IGNORE;IGNORE % TAG LATIN SMALL LETTER F
-<U000E0067> IGNORE;IGNORE;IGNORE;IGNORE % TAG LATIN SMALL LETTER G
-<U000E0068> IGNORE;IGNORE;IGNORE;IGNORE % TAG LATIN SMALL LETTER H
-<U000E0069> IGNORE;IGNORE;IGNORE;IGNORE % TAG LATIN SMALL LETTER I
-<U000E006A> IGNORE;IGNORE;IGNORE;IGNORE % TAG LATIN SMALL LETTER J
-<U000E006B> IGNORE;IGNORE;IGNORE;IGNORE % TAG LATIN SMALL LETTER K
-<U000E006C> IGNORE;IGNORE;IGNORE;IGNORE % TAG LATIN SMALL LETTER L
-<U000E006D> IGNORE;IGNORE;IGNORE;IGNORE % TAG LATIN SMALL LETTER M
-<U000E006E> IGNORE;IGNORE;IGNORE;IGNORE % TAG LATIN SMALL LETTER N
-<U000E006F> IGNORE;IGNORE;IGNORE;IGNORE % TAG LATIN SMALL LETTER O
-<U000E0070> IGNORE;IGNORE;IGNORE;IGNORE % TAG LATIN SMALL LETTER P
-<U000E0071> IGNORE;IGNORE;IGNORE;IGNORE % TAG LATIN SMALL LETTER Q
-<U000E0072> IGNORE;IGNORE;IGNORE;IGNORE % TAG LATIN SMALL LETTER R
-<U000E0073> IGNORE;IGNORE;IGNORE;IGNORE % TAG LATIN SMALL LETTER S
-<U000E0074> IGNORE;IGNORE;IGNORE;IGNORE % TAG LATIN SMALL LETTER T
-<U000E0075> IGNORE;IGNORE;IGNORE;IGNORE % TAG LATIN SMALL LETTER U
-<U000E0076> IGNORE;IGNORE;IGNORE;IGNORE % TAG LATIN SMALL LETTER V
-<U000E0077> IGNORE;IGNORE;IGNORE;IGNORE % TAG LATIN SMALL LETTER W
-<U000E0078> IGNORE;IGNORE;IGNORE;IGNORE % TAG LATIN SMALL LETTER X
-<U000E0079> IGNORE;IGNORE;IGNORE;IGNORE % TAG LATIN SMALL LETTER Y
-<U000E007A> IGNORE;IGNORE;IGNORE;IGNORE % TAG LATIN SMALL LETTER Z
-<U000E007B> IGNORE;IGNORE;IGNORE;IGNORE % TAG LEFT CURLY BRACKET
-<U000E007C> IGNORE;IGNORE;IGNORE;IGNORE % TAG VERTICAL LINE
-<U000E007D> IGNORE;IGNORE;IGNORE;IGNORE % TAG RIGHT CURLY BRACKET
-<U000E007E> IGNORE;IGNORE;IGNORE;IGNORE % TAG TILDE
-<U000E007F> IGNORE;IGNORE;IGNORE;IGNORE % CANCEL TAG
-<U000E0100> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-17
-<U000E0101> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-18
-<U000E0102> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-19
-<U000E0103> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-20
-<U000E0104> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-21
-<U000E0105> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-22
-<U000E0106> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-23
-<U000E0107> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-24
-<U000E0108> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-25
-<U000E0109> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-26
-<U000E010A> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-27
-<U000E010B> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-28
-<U000E010C> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-29
-<U000E010D> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-30
-<U000E010E> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-31
-<U000E010F> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-32
-<U000E0110> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-33
-<U000E0111> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-34
-<U000E0112> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-35
-<U000E0113> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-36
-<U000E0114> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-37
-<U000E0115> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-38
-<U000E0116> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-39
-<U000E0117> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-40
-<U000E0118> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-41
-<U000E0119> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-42
-<U000E011A> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-43
-<U000E011B> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-44
-<U000E011C> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-45
-<U000E011D> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-46
-<U000E011E> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-47
-<U000E011F> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-48
-<U000E0120> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-49
-<U000E0121> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-50
-<U000E0122> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-51
-<U000E0123> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-52
-<U000E0124> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-53
-<U000E0125> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-54
-<U000E0126> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-55
-<U000E0127> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-56
-<U000E0128> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-57
-<U000E0129> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-58
-<U000E012A> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-59
-<U000E012B> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-60
-<U000E012C> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-61
-<U000E012D> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-62
-<U000E012E> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-63
-<U000E012F> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-64
-<U000E0130> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-65
-<U000E0131> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-66
-<U000E0132> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-67
-<U000E0133> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-68
-<U000E0134> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-69
-<U000E0135> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-70
-<U000E0136> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-71
-<U000E0137> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-72
-<U000E0138> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-73
-<U000E0139> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-74
-<U000E013A> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-75
-<U000E013B> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-76
-<U000E013C> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-77
-<U000E013D> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-78
-<U000E013E> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-79
-<U000E013F> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-80
-<U000E0140> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-81
-<U000E0141> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-82
-<U000E0142> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-83
-<U000E0143> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-84
-<U000E0144> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-85
-<U000E0145> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-86
-<U000E0146> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-87
-<U000E0147> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-88
-<U000E0148> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-89
-<U000E0149> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-90
-<U000E014A> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-91
-<U000E014B> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-92
-<U000E014C> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-93
-<U000E014D> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-94
-<U000E014E> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-95
-<U000E014F> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-96
-<U000E0150> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-97
-<U000E0151> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-98
-<U000E0152> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-99
-<U000E0153> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-100
-<U000E0154> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-101
-<U000E0155> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-102
-<U000E0156> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-103
-<U000E0157> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-104
-<U000E0158> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-105
-<U000E0159> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-106
-<U000E015A> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-107
-<U000E015B> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-108
-<U000E015C> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-109
-<U000E015D> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-110
-<U000E015E> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-111
-<U000E015F> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-112
-<U000E0160> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-113
-<U000E0161> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-114
-<U000E0162> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-115
-<U000E0163> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-116
-<U000E0164> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-117
-<U000E0165> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-118
-<U000E0166> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-119
-<U000E0167> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-120
-<U000E0168> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-121
-<U000E0169> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-122
-<U000E016A> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-123
-<U000E016B> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-124
-<U000E016C> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-125
-<U000E016D> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-126
-<U000E016E> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-127
-<U000E016F> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-128
-<U000E0170> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-129
-<U000E0171> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-130
-<U000E0172> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-131
-<U000E0173> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-132
-<U000E0174> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-133
-<U000E0175> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-134
-<U000E0176> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-135
-<U000E0177> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-136
-<U000E0178> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-137
-<U000E0179> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-138
-<U000E017A> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-139
-<U000E017B> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-140
-<U000E017C> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-141
-<U000E017D> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-142
-<U000E017E> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-143
-<U000E017F> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-144
-<U000E0180> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-145
-<U000E0181> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-146
-<U000E0182> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-147
-<U000E0183> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-148
-<U000E0184> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-149
-<U000E0185> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-150
-<U000E0186> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-151
-<U000E0187> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-152
-<U000E0188> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-153
-<U000E0189> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-154
-<U000E018A> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-155
-<U000E018B> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-156
-<U000E018C> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-157
-<U000E018D> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-158
-<U000E018E> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-159
-<U000E018F> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-160
-<U000E0190> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-161
-<U000E0191> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-162
-<U000E0192> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-163
-<U000E0193> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-164
-<U000E0194> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-165
-<U000E0195> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-166
-<U000E0196> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-167
-<U000E0197> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-168
-<U000E0198> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-169
-<U000E0199> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-170
-<U000E019A> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-171
-<U000E019B> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-172
-<U000E019C> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-173
-<U000E019D> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-174
-<U000E019E> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-175
-<U000E019F> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-176
-<U000E01A0> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-177
-<U000E01A1> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-178
-<U000E01A2> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-179
-<U000E01A3> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-180
-<U000E01A4> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-181
-<U000E01A5> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-182
-<U000E01A6> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-183
-<U000E01A7> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-184
-<U000E01A8> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-185
-<U000E01A9> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-186
-<U000E01AA> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-187
-<U000E01AB> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-188
-<U000E01AC> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-189
-<U000E01AD> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-190
-<U000E01AE> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-191
-<U000E01AF> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-192
-<U000E01B0> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-193
-<U000E01B1> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-194
-<U000E01B2> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-195
-<U000E01B3> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-196
-<U000E01B4> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-197
-<U000E01B5> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-198
-<U000E01B6> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-199
-<U000E01B7> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-200
-<U000E01B8> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-201
-<U000E01B9> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-202
-<U000E01BA> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-203
-<U000E01BB> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-204
-<U000E01BC> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-205
-<U000E01BD> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-206
-<U000E01BE> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-207
-<U000E01BF> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-208
-<U000E01C0> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-209
-<U000E01C1> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-210
-<U000E01C2> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-211
-<U000E01C3> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-212
-<U000E01C4> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-213
-<U000E01C5> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-214
-<U000E01C6> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-215
-<U000E01C7> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-216
-<U000E01C8> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-217
-<U000E01C9> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-218
-<U000E01CA> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-219
-<U000E01CB> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-220
-<U000E01CC> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-221
-<U000E01CD> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-222
-<U000E01CE> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-223
-<U000E01CF> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-224
-<U000E01D0> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-225
-<U000E01D1> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-226
-<U000E01D2> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-227
-<U000E01D3> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-228
-<U000E01D4> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-229
-<U000E01D5> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-230
-<U000E01D6> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-231
-<U000E01D7> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-232
-<U000E01D8> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-233
-<U000E01D9> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-234
-<U000E01DA> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-235
-<U000E01DB> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-236
-<U000E01DC> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-237
-<U000E01DD> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-238
-<U000E01DE> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-239
-<U000E01DF> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-240
-<U000E01E0> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-241
-<U000E01E1> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-242
-<U000E01E2> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-243
-<U000E01E3> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-244
-<U000E01E4> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-245
-<U000E01E5> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-246
-<U000E01E6> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-247
-<U000E01E7> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-248
-<U000E01E8> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-249
-<U000E01E9> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-250
-<U000E01EA> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-251
-<U000E01EB> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-252
-<U000E01EC> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-253
-<U000E01ED> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-254
-<U000E01EE> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-255
-<U000E01EF> IGNORE;IGNORE;IGNORE;IGNORE % VARIATION SELECTOR-256
+<U0000> IGNORE;IGNORE;IGNORE;<U0000> % NULL (in ISO 6429)
+<U0001> IGNORE;IGNORE;IGNORE;<U0001> % START OF HEADING (in ISO 6429)
+<U0002> IGNORE;IGNORE;IGNORE;<U0002> % START OF TEXT (in ISO 6429)
+<U0003> IGNORE;IGNORE;IGNORE;<U0003> % END OF TEXT (in ISO 6429)
+<U0004> IGNORE;IGNORE;IGNORE;<U0004> % END OF TRANSMISSION (in ISO 6429)
+<U0005> IGNORE;IGNORE;IGNORE;<U0005> % ENQUIRY (in ISO 6429)
+<U0006> IGNORE;IGNORE;IGNORE;<U0006> % ACKNOWLEDGE (in ISO 6429)
+<U0007> IGNORE;IGNORE;IGNORE;<U0007> % BELL (in ISO 6429)
+<U0008> IGNORE;IGNORE;IGNORE;<U0008> % BACKSPACE (in ISO 6429)
+<U000E> IGNORE;IGNORE;IGNORE;<U000E> % SHIFT OUT (in ISO 6429)
+<U000F> IGNORE;IGNORE;IGNORE;<U000F> % SHIFT IN (in ISO 6429)
+<U0010> IGNORE;IGNORE;IGNORE;<U0010> % DATA LINK ESCAPE (in ISO 6429)
+<U0011> IGNORE;IGNORE;IGNORE;<U0011> % DEVICE CONTROL ONE (in ISO 6429)
+<U0012> IGNORE;IGNORE;IGNORE;<U0012> % DEVICE CONTROL TWO (in ISO 6429)
+<U0013> IGNORE;IGNORE;IGNORE;<U0013> % DEVICE CONTROL THREE (in ISO 6429)
+<U0014> IGNORE;IGNORE;IGNORE;<U0014> % DEVICE CONTROL FOUR (in ISO 6429)
+<U0015> IGNORE;IGNORE;IGNORE;<U0015> % NEGATIVE ACKNOWLEDGE (in ISO 6429)
+<U0016> IGNORE;IGNORE;IGNORE;<U0016> % SYNCHRONOUS IDLE (in ISO 6429)
+<U0017> IGNORE;IGNORE;IGNORE;<U0017> % END OF TRANSMISSION BLOCK (in ISO 6429)
+<U0018> IGNORE;IGNORE;IGNORE;<U0018> % CANCEL (in ISO 6429)
+<U0019> IGNORE;IGNORE;IGNORE;<U0019> % END OF MEDIUM (in ISO 6429)
+<U001A> IGNORE;IGNORE;IGNORE;<U001A> % SUBSTITUTE (in ISO 6429)
+<U001B> IGNORE;IGNORE;IGNORE;<U001B> % ESCAPE (in ISO 6429)
+<U001C> IGNORE;IGNORE;IGNORE;<U001C> % FILE SEPARATOR (in ISO 6429)
+<U001D> IGNORE;IGNORE;IGNORE;<U001D> % GROUP SEPARATOR (in ISO 6429)
+<U001E> IGNORE;IGNORE;IGNORE;<U001E> % RECORD SEPARATOR (in ISO 6429)
+<U001F> IGNORE;IGNORE;IGNORE;<U001F> % UNIT SEPARATOR (in ISO 6429)
+<U007F> IGNORE;IGNORE;IGNORE;<U007F> % DELETE (in ISO 6429)
+<U0080> IGNORE;IGNORE;IGNORE;<U0080> % <control>
+<U0081> IGNORE;IGNORE;IGNORE;<U0081> % <control>
+<U0082> IGNORE;IGNORE;IGNORE;<U0082> % BREAK PERMITTED HERE (in ISO 6429)
+<U0083> IGNORE;IGNORE;IGNORE;<U0083> % NO BREAK HERE (in ISO 6429)
+<U0084> IGNORE;IGNORE;IGNORE;<U0084> % <control>
+<U0086> IGNORE;IGNORE;IGNORE;<U0086> % START OF SELECTED AREA (in ISO 6429)
+<U0087> IGNORE;IGNORE;IGNORE;<U0087> % END OF SELECTED AREA (in ISO 6429)
+<U0088> IGNORE;IGNORE;IGNORE;<U0088> % CHARACTER TABULATION SET (in ISO 6429)
+<U0089> IGNORE;IGNORE;IGNORE;<U0089> % CHARACTER TABULATION WITH JUSTIFICATION (in ISO 6429)
+<U008A> IGNORE;IGNORE;IGNORE;<U008A> % LINE TABULATION SET (in ISO 6429)
+<U008B> IGNORE;IGNORE;IGNORE;<U008B> % PARTIAL LINE FORWARD (in ISO 6429)
+<U008C> IGNORE;IGNORE;IGNORE;<U008C> % PARTIAL LINE BACKWARD (in ISO 6429)
+<U008D> IGNORE;IGNORE;IGNORE;<U008D> % PARTIAL LINE FEED (in ISO 6429)
+<U008E> IGNORE;IGNORE;IGNORE;<U008E> % SINGLE SHIFT TWO (in ISO 6429)
+<U008F> IGNORE;IGNORE;IGNORE;<U008F> % SINGLE SHIFT THREE (in ISO 6429)
+<U0090> IGNORE;IGNORE;IGNORE;<U0090> % DEVICE CONTROL STRING (in ISO 6429)
+<U0091> IGNORE;IGNORE;IGNORE;<U0091> % PRIVATE USE ONE (in ISO 6429)
+<U0092> IGNORE;IGNORE;IGNORE;<U0092> % PRIVATE USE TWO (in ISO 6429)
+<U0093> IGNORE;IGNORE;IGNORE;<U0093> % SET TRANSMIT STATE (in ISO 6429)
+<U0094> IGNORE;IGNORE;IGNORE;<U0094> % CANCEL CHARACTER (in ISO 6429)
+<U0095> IGNORE;IGNORE;IGNORE;<U0095> % MESSAGE WAITING (in ISO 6429)
+<U0096> IGNORE;IGNORE;IGNORE;<U0096> % START OF GUARDED AREA (in ISO 6429)
+<U0097> IGNORE;IGNORE;IGNORE;<U0097> % END OF GUARDED AREA (in ISO 6429)
+<U0098> IGNORE;IGNORE;IGNORE;<U0098> % START OF STRING (in ISO 6429)
+<U0099> IGNORE;IGNORE;IGNORE;<U0099> % <control>
+<U009A> IGNORE;IGNORE;IGNORE;<U009A> % SINGLE CHARACTER INTRODUCER (in ISO 6429)
+<U009B> IGNORE;IGNORE;IGNORE;<U009B> % CONTROL SEQUENCE INTRODUCER (in ISO 6429)
+<U009C> IGNORE;IGNORE;IGNORE;<U009C> % STRING TERMINATOR (in ISO 6429)
+<U009D> IGNORE;IGNORE;IGNORE;<U009D> % OPERATING SYSTEM COMMAND (in ISO 6429)
+<U009E> IGNORE;IGNORE;IGNORE;<U009E> % PRIVACY MESSAGE (in ISO 6429)
+<U009F> IGNORE;IGNORE;IGNORE;<U009F> % APPLICATION PROGRAM COMMAND (in ISO 6429)
+<U00AD> IGNORE;IGNORE;IGNORE;<U00AD> % SOFT HYPHEN
+<U061C> IGNORE;IGNORE;IGNORE;<U061C> % ARABIC LETTER MARK
+<U070F> IGNORE;IGNORE;IGNORE;<U070F> % SYRIAC ABBREVIATION MARK
+<U08E2> IGNORE;IGNORE;IGNORE;<U08E2> % ARABIC DISPUTED END OF AYAH
+<U180B> IGNORE;IGNORE;IGNORE;<U180B> % MONGOLIAN FREE VARIATION SELECTOR ONE
+<U180C> IGNORE;IGNORE;IGNORE;<U180C> % MONGOLIAN FREE VARIATION SELECTOR TWO
+<U180D> IGNORE;IGNORE;IGNORE;<U180D> % MONGOLIAN FREE VARIATION SELECTOR THREE
+<U180E> IGNORE;IGNORE;IGNORE;<U180E> % MONGOLIAN VOWEL SEPARATOR
+<U200B> IGNORE;IGNORE;IGNORE;<U200B> % ZERO WIDTH SPACE
+<U200C> IGNORE;IGNORE;IGNORE;<U200C> % ZERO WIDTH NON-JOINER
+<U200D> IGNORE;IGNORE;IGNORE;<U200D> % ZERO WIDTH JOINER
+<U200E> IGNORE;IGNORE;IGNORE;<U200E> % LEFT-TO-RIGHT MARK
+<U200F> IGNORE;IGNORE;IGNORE;<U200F> % RIGHT-TO-LEFT MARK
+<U202A> IGNORE;IGNORE;IGNORE;<U202A> % LEFT-TO-RIGHT EMBEDDING
+<U202B> IGNORE;IGNORE;IGNORE;<U202B> % RIGHT-TO-LEFT EMBEDDING
+<U202C> IGNORE;IGNORE;IGNORE;<U202C> % POP DIRECTIONAL FORMATTING
+<U202D> IGNORE;IGNORE;IGNORE;<U202D> % LEFT-TO-RIGHT OVERRIDE
+<U202E> IGNORE;IGNORE;IGNORE;<U202E> % RIGHT-TO-LEFT OVERRIDE
+<U2060> IGNORE;IGNORE;IGNORE;<U2060> % WORD JOINER
+<U2066> IGNORE;IGNORE;IGNORE;<U2066> % LEFT-TO-RIGHT ISOLATE
+<U2067> IGNORE;IGNORE;IGNORE;<U2067> % RIGHT-TO-LEFT ISOLATE
+<U2068> IGNORE;IGNORE;IGNORE;<U2068> % FIRST STRONG ISOLATE
+<U2069> IGNORE;IGNORE;IGNORE;<U2069> % POP DIRECTIONAL ISOLATE
+<U206A> IGNORE;IGNORE;IGNORE;<U206A> % INHIBIT SYMMETRIC SWAPPING
+<U206B> IGNORE;IGNORE;IGNORE;<U206B> % ACTIVATE SYMMETRIC SWAPPING
+<U206C> IGNORE;IGNORE;IGNORE;<U206C> % INHIBIT ARABIC FORM SHAPING
+<U206D> IGNORE;IGNORE;IGNORE;<U206D> % ACTIVATE ARABIC FORM SHAPING
+<U206E> IGNORE;IGNORE;IGNORE;<U206E> % NATIONAL DIGIT SHAPES
+<U206F> IGNORE;IGNORE;IGNORE;<U206F> % NOMINAL DIGIT SHAPES
+<UFE00> IGNORE;IGNORE;IGNORE;<UFE00> % VARIATION SELECTOR-1
+<UFE01> IGNORE;IGNORE;IGNORE;<UFE01> % VARIATION SELECTOR-2
+<UFE02> IGNORE;IGNORE;IGNORE;<UFE02> % VARIATION SELECTOR-3
+<UFE03> IGNORE;IGNORE;IGNORE;<UFE03> % VARIATION SELECTOR-4
+<UFE04> IGNORE;IGNORE;IGNORE;<UFE04> % VARIATION SELECTOR-5
+<UFE05> IGNORE;IGNORE;IGNORE;<UFE05> % VARIATION SELECTOR-6
+<UFE06> IGNORE;IGNORE;IGNORE;<UFE06> % VARIATION SELECTOR-7
+<UFE07> IGNORE;IGNORE;IGNORE;<UFE07> % VARIATION SELECTOR-8
+<UFE08> IGNORE;IGNORE;IGNORE;<UFE08> % VARIATION SELECTOR-9
+<UFE09> IGNORE;IGNORE;IGNORE;<UFE09> % VARIATION SELECTOR-10
+<UFE0A> IGNORE;IGNORE;IGNORE;<UFE0A> % VARIATION SELECTOR-11
+<UFE0B> IGNORE;IGNORE;IGNORE;<UFE0B> % VARIATION SELECTOR-12
+<UFE0C> IGNORE;IGNORE;IGNORE;<UFE0C> % VARIATION SELECTOR-13
+<UFE0D> IGNORE;IGNORE;IGNORE;<UFE0D> % VARIATION SELECTOR-14
+<UFE0E> IGNORE;IGNORE;IGNORE;<UFE0E> % VARIATION SELECTOR-15
+<UFE0F> IGNORE;IGNORE;IGNORE;<UFE0F> % VARIATION SELECTOR-16
+<UFEFF> IGNORE;IGNORE;IGNORE;<UFEFF> % ZERO WIDTH NO-BREAK SPACE
+<UFFF9> IGNORE;IGNORE;IGNORE;<UFFF9> % INTERLINEAR ANNOTATION ANCHOR
+<UFFFA> IGNORE;IGNORE;IGNORE;<UFFFA> % INTERLINEAR ANNOTATION SEPARATOR
+<UFFFB> IGNORE;IGNORE;IGNORE;<UFFFB> % INTERLINEAR ANNOTATION TERMINATOR
+<U0001BCA0> IGNORE;IGNORE;IGNORE;<U0001BCA0> % SHORTHAND FORMAT LETTER OVERLAP
+<U0001BCA1> IGNORE;IGNORE;IGNORE;<U0001BCA1> % SHORTHAND FORMAT CONTINUING OVERLAP
+<U0001BCA2> IGNORE;IGNORE;IGNORE;<U0001BCA2> % SHORTHAND FORMAT DOWN STEP
+<U0001BCA3> IGNORE;IGNORE;IGNORE;<U0001BCA3> % SHORTHAND FORMAT UP STEP
+<U0001D173> IGNORE;IGNORE;IGNORE;<U0001D173> % MUSICAL SYMBOL BEGIN BEAM
+<U0001D174> IGNORE;IGNORE;IGNORE;<U0001D174> % MUSICAL SYMBOL END BEAM
+<U0001D175> IGNORE;IGNORE;IGNORE;<U0001D175> % MUSICAL SYMBOL BEGIN TIE
+<U0001D176> IGNORE;IGNORE;IGNORE;<U0001D176> % MUSICAL SYMBOL END TIE
+<U0001D177> IGNORE;IGNORE;IGNORE;<U0001D177> % MUSICAL SYMBOL BEGIN SLUR
+<U0001D178> IGNORE;IGNORE;IGNORE;<U0001D178> % MUSICAL SYMBOL END SLUR
+<U0001D179> IGNORE;IGNORE;IGNORE;<U0001D179> % MUSICAL SYMBOL BEGIN PHRASE
+<U0001D17A> IGNORE;IGNORE;IGNORE;<U0001D17A> % MUSICAL SYMBOL END PHRASE
+<U000E0001> IGNORE;IGNORE;IGNORE;<U000E0001> % LANGUAGE TAG
+<U000E0020> IGNORE;IGNORE;IGNORE;<U000E0020> % TAG SPACE
+<U000E0021> IGNORE;IGNORE;IGNORE;<U000E0021> % TAG EXCLAMATION MARK
+<U000E0022> IGNORE;IGNORE;IGNORE;<U000E0022> % TAG QUOTATION MARK
+<U000E0023> IGNORE;IGNORE;IGNORE;<U000E0023> % TAG NUMBER SIGN
+<U000E0024> IGNORE;IGNORE;IGNORE;<U000E0024> % TAG DOLLAR SIGN
+<U000E0025> IGNORE;IGNORE;IGNORE;<U000E0025> % TAG PERCENT SIGN
+<U000E0026> IGNORE;IGNORE;IGNORE;<U000E0026> % TAG AMPERSAND
+<U000E0027> IGNORE;IGNORE;IGNORE;<U000E0027> % TAG APOSTROPHE
+<U000E0028> IGNORE;IGNORE;IGNORE;<U000E0028> % TAG LEFT PARENTHESIS
+<U000E0029> IGNORE;IGNORE;IGNORE;<U000E0029> % TAG RIGHT PARENTHESIS
+<U000E002A> IGNORE;IGNORE;IGNORE;<U000E002A> % TAG ASTERISK
+<U000E002B> IGNORE;IGNORE;IGNORE;<U000E002B> % TAG PLUS SIGN
+<U000E002C> IGNORE;IGNORE;IGNORE;<U000E002C> % TAG COMMA
+<U000E002D> IGNORE;IGNORE;IGNORE;<U000E002D> % TAG HYPHEN-MINUS
+<U000E002E> IGNORE;IGNORE;IGNORE;<U000E002E> % TAG FULL STOP
+<U000E002F> IGNORE;IGNORE;IGNORE;<U000E002F> % TAG SOLIDUS
+<U000E0030> IGNORE;IGNORE;IGNORE;<U000E0030> % TAG DIGIT ZERO
+<U000E0031> IGNORE;IGNORE;IGNORE;<U000E0031> % TAG DIGIT ONE
+<U000E0032> IGNORE;IGNORE;IGNORE;<U000E0032> % TAG DIGIT TWO
+<U000E0033> IGNORE;IGNORE;IGNORE;<U000E0033> % TAG DIGIT THREE
+<U000E0034> IGNORE;IGNORE;IGNORE;<U000E0034> % TAG DIGIT FOUR
+<U000E0035> IGNORE;IGNORE;IGNORE;<U000E0035> % TAG DIGIT FIVE
+<U000E0036> IGNORE;IGNORE;IGNORE;<U000E0036> % TAG DIGIT SIX
+<U000E0037> IGNORE;IGNORE;IGNORE;<U000E0037> % TAG DIGIT SEVEN
+<U000E0038> IGNORE;IGNORE;IGNORE;<U000E0038> % TAG DIGIT EIGHT
+<U000E0039> IGNORE;IGNORE;IGNORE;<U000E0039> % TAG DIGIT NINE
+<U000E003A> IGNORE;IGNORE;IGNORE;<U000E003A> % TAG COLON
+<U000E003B> IGNORE;IGNORE;IGNORE;<U000E003B> % TAG SEMICOLON
+<U000E003C> IGNORE;IGNORE;IGNORE;<U000E003C> % TAG LESS-THAN SIGN
+<U000E003D> IGNORE;IGNORE;IGNORE;<U000E003D> % TAG EQUALS SIGN
+<U000E003E> IGNORE;IGNORE;IGNORE;<U000E003E> % TAG GREATER-THAN SIGN
+<U000E003F> IGNORE;IGNORE;IGNORE;<U000E003F> % TAG QUESTION MARK
+<U000E0040> IGNORE;IGNORE;IGNORE;<U000E0040> % TAG COMMERCIAL AT
+<U000E0041> IGNORE;IGNORE;IGNORE;<U000E0041> % TAG LATIN CAPITAL LETTER A
+<U000E0042> IGNORE;IGNORE;IGNORE;<U000E0042> % TAG LATIN CAPITAL LETTER B
+<U000E0043> IGNORE;IGNORE;IGNORE;<U000E0043> % TAG LATIN CAPITAL LETTER C
+<U000E0044> IGNORE;IGNORE;IGNORE;<U000E0044> % TAG LATIN CAPITAL LETTER D
+<U000E0045> IGNORE;IGNORE;IGNORE;<U000E0045> % TAG LATIN CAPITAL LETTER E
+<U000E0046> IGNORE;IGNORE;IGNORE;<U000E0046> % TAG LATIN CAPITAL LETTER F
+<U000E0047> IGNORE;IGNORE;IGNORE;<U000E0047> % TAG LATIN CAPITAL LETTER G
+<U000E0048> IGNORE;IGNORE;IGNORE;<U000E0048> % TAG LATIN CAPITAL LETTER H
+<U000E0049> IGNORE;IGNORE;IGNORE;<U000E0049> % TAG LATIN CAPITAL LETTER I
+<U000E004A> IGNORE;IGNORE;IGNORE;<U000E004A> % TAG LATIN CAPITAL LETTER J
+<U000E004B> IGNORE;IGNORE;IGNORE;<U000E004B> % TAG LATIN CAPITAL LETTER K
+<U000E004C> IGNORE;IGNORE;IGNORE;<U000E004C> % TAG LATIN CAPITAL LETTER L
+<U000E004D> IGNORE;IGNORE;IGNORE;<U000E004D> % TAG LATIN CAPITAL LETTER M
+<U000E004E> IGNORE;IGNORE;IGNORE;<U000E004E> % TAG LATIN CAPITAL LETTER N
+<U000E004F> IGNORE;IGNORE;IGNORE;<U000E004F> % TAG LATIN CAPITAL LETTER O
+<U000E0050> IGNORE;IGNORE;IGNORE;<U000E0050> % TAG LATIN CAPITAL LETTER P
+<U000E0051> IGNORE;IGNORE;IGNORE;<U000E0051> % TAG LATIN CAPITAL LETTER Q
+<U000E0052> IGNORE;IGNORE;IGNORE;<U000E0052> % TAG LATIN CAPITAL LETTER R
+<U000E0053> IGNORE;IGNORE;IGNORE;<U000E0053> % TAG LATIN CAPITAL LETTER S
+<U000E0054> IGNORE;IGNORE;IGNORE;<U000E0054> % TAG LATIN CAPITAL LETTER T
+<U000E0055> IGNORE;IGNORE;IGNORE;<U000E0055> % TAG LATIN CAPITAL LETTER U
+<U000E0056> IGNORE;IGNORE;IGNORE;<U000E0056> % TAG LATIN CAPITAL LETTER V
+<U000E0057> IGNORE;IGNORE;IGNORE;<U000E0057> % TAG LATIN CAPITAL LETTER W
+<U000E0058> IGNORE;IGNORE;IGNORE;<U000E0058> % TAG LATIN CAPITAL LETTER X
+<U000E0059> IGNORE;IGNORE;IGNORE;<U000E0059> % TAG LATIN CAPITAL LETTER Y
+<U000E005A> IGNORE;IGNORE;IGNORE;<U000E005A> % TAG LATIN CAPITAL LETTER Z
+<U000E005B> IGNORE;IGNORE;IGNORE;<U000E005B> % TAG LEFT SQUARE BRACKET
+<U000E005C> IGNORE;IGNORE;IGNORE;<U000E005C> % TAG REVERSE SOLIDUS
+<U000E005D> IGNORE;IGNORE;IGNORE;<U000E005D> % TAG RIGHT SQUARE BRACKET
+<U000E005E> IGNORE;IGNORE;IGNORE;<U000E005E> % TAG CIRCUMFLEX ACCENT
+<U000E005F> IGNORE;IGNORE;IGNORE;<U000E005F> % TAG LOW LINE
+<U000E0060> IGNORE;IGNORE;IGNORE;<U000E0060> % TAG GRAVE ACCENT
+<U000E0061> IGNORE;IGNORE;IGNORE;<U000E0061> % TAG LATIN SMALL LETTER A
+<U000E0062> IGNORE;IGNORE;IGNORE;<U000E0062> % TAG LATIN SMALL LETTER B
+<U000E0063> IGNORE;IGNORE;IGNORE;<U000E0063> % TAG LATIN SMALL LETTER C
+<U000E0064> IGNORE;IGNORE;IGNORE;<U000E0064> % TAG LATIN SMALL LETTER D
+<U000E0065> IGNORE;IGNORE;IGNORE;<U000E0065> % TAG LATIN SMALL LETTER E
+<U000E0066> IGNORE;IGNORE;IGNORE;<U000E0066> % TAG LATIN SMALL LETTER F
+<U000E0067> IGNORE;IGNORE;IGNORE;<U000E0067> % TAG LATIN SMALL LETTER G
+<U000E0068> IGNORE;IGNORE;IGNORE;<U000E0068> % TAG LATIN SMALL LETTER H
+<U000E0069> IGNORE;IGNORE;IGNORE;<U000E0069> % TAG LATIN SMALL LETTER I
+<U000E006A> IGNORE;IGNORE;IGNORE;<U000E006A> % TAG LATIN SMALL LETTER J
+<U000E006B> IGNORE;IGNORE;IGNORE;<U000E006B> % TAG LATIN SMALL LETTER K
+<U000E006C> IGNORE;IGNORE;IGNORE;<U000E006C> % TAG LATIN SMALL LETTER L
+<U000E006D> IGNORE;IGNORE;IGNORE;<U000E006D> % TAG LATIN SMALL LETTER M
+<U000E006E> IGNORE;IGNORE;IGNORE;<U000E006E> % TAG LATIN SMALL LETTER N
+<U000E006F> IGNORE;IGNORE;IGNORE;<U000E006F> % TAG LATIN SMALL LETTER O
+<U000E0070> IGNORE;IGNORE;IGNORE;<U000E0070> % TAG LATIN SMALL LETTER P
+<U000E0071> IGNORE;IGNORE;IGNORE;<U000E0071> % TAG LATIN SMALL LETTER Q
+<U000E0072> IGNORE;IGNORE;IGNORE;<U000E0072> % TAG LATIN SMALL LETTER R
+<U000E0073> IGNORE;IGNORE;IGNORE;<U000E0073> % TAG LATIN SMALL LETTER S
+<U000E0074> IGNORE;IGNORE;IGNORE;<U000E0074> % TAG LATIN SMALL LETTER T
+<U000E0075> IGNORE;IGNORE;IGNORE;<U000E0075> % TAG LATIN SMALL LETTER U
+<U000E0076> IGNORE;IGNORE;IGNORE;<U000E0076> % TAG LATIN SMALL LETTER V
+<U000E0077> IGNORE;IGNORE;IGNORE;<U000E0077> % TAG LATIN SMALL LETTER W
+<U000E0078> IGNORE;IGNORE;IGNORE;<U000E0078> % TAG LATIN SMALL LETTER X
+<U000E0079> IGNORE;IGNORE;IGNORE;<U000E0079> % TAG LATIN SMALL LETTER Y
+<U000E007A> IGNORE;IGNORE;IGNORE;<U000E007A> % TAG LATIN SMALL LETTER Z
+<U000E007B> IGNORE;IGNORE;IGNORE;<U000E007B> % TAG LEFT CURLY BRACKET
+<U000E007C> IGNORE;IGNORE;IGNORE;<U000E007C> % TAG VERTICAL LINE
+<U000E007D> IGNORE;IGNORE;IGNORE;<U000E007D> % TAG RIGHT CURLY BRACKET
+<U000E007E> IGNORE;IGNORE;IGNORE;<U000E007E> % TAG TILDE
+<U000E007F> IGNORE;IGNORE;IGNORE;<U000E007F> % CANCEL TAG
+<U000E0100> IGNORE;IGNORE;IGNORE;<U000E0100> % VARIATION SELECTOR-17
+<U000E0101> IGNORE;IGNORE;IGNORE;<U000E0101> % VARIATION SELECTOR-18
+<U000E0102> IGNORE;IGNORE;IGNORE;<U000E0102> % VARIATION SELECTOR-19
+<U000E0103> IGNORE;IGNORE;IGNORE;<U000E0103> % VARIATION SELECTOR-20
+<U000E0104> IGNORE;IGNORE;IGNORE;<U000E0104> % VARIATION SELECTOR-21
+<U000E0105> IGNORE;IGNORE;IGNORE;<U000E0105> % VARIATION SELECTOR-22
+<U000E0106> IGNORE;IGNORE;IGNORE;<U000E0106> % VARIATION SELECTOR-23
+<U000E0107> IGNORE;IGNORE;IGNORE;<U000E0107> % VARIATION SELECTOR-24
+<U000E0108> IGNORE;IGNORE;IGNORE;<U000E0108> % VARIATION SELECTOR-25
+<U000E0109> IGNORE;IGNORE;IGNORE;<U000E0109> % VARIATION SELECTOR-26
+<U000E010A> IGNORE;IGNORE;IGNORE;<U000E010A> % VARIATION SELECTOR-27
+<U000E010B> IGNORE;IGNORE;IGNORE;<U000E010B> % VARIATION SELECTOR-28
+<U000E010C> IGNORE;IGNORE;IGNORE;<U000E010C> % VARIATION SELECTOR-29
+<U000E010D> IGNORE;IGNORE;IGNORE;<U000E010D> % VARIATION SELECTOR-30
+<U000E010E> IGNORE;IGNORE;IGNORE;<U000E010E> % VARIATION SELECTOR-31
+<U000E010F> IGNORE;IGNORE;IGNORE;<U000E010F> % VARIATION SELECTOR-32
+<U000E0110> IGNORE;IGNORE;IGNORE;<U000E0110> % VARIATION SELECTOR-33
+<U000E0111> IGNORE;IGNORE;IGNORE;<U000E0111> % VARIATION SELECTOR-34
+<U000E0112> IGNORE;IGNORE;IGNORE;<U000E0112> % VARIATION SELECTOR-35
+<U000E0113> IGNORE;IGNORE;IGNORE;<U000E0113> % VARIATION SELECTOR-36
+<U000E0114> IGNORE;IGNORE;IGNORE;<U000E0114> % VARIATION SELECTOR-37
+<U000E0115> IGNORE;IGNORE;IGNORE;<U000E0115> % VARIATION SELECTOR-38
+<U000E0116> IGNORE;IGNORE;IGNORE;<U000E0116> % VARIATION SELECTOR-39
+<U000E0117> IGNORE;IGNORE;IGNORE;<U000E0117> % VARIATION SELECTOR-40
+<U000E0118> IGNORE;IGNORE;IGNORE;<U000E0118> % VARIATION SELECTOR-41
+<U000E0119> IGNORE;IGNORE;IGNORE;<U000E0119> % VARIATION SELECTOR-42
+<U000E011A> IGNORE;IGNORE;IGNORE;<U000E011A> % VARIATION SELECTOR-43
+<U000E011B> IGNORE;IGNORE;IGNORE;<U000E011B> % VARIATION SELECTOR-44
+<U000E011C> IGNORE;IGNORE;IGNORE;<U000E011C> % VARIATION SELECTOR-45
+<U000E011D> IGNORE;IGNORE;IGNORE;<U000E011D> % VARIATION SELECTOR-46
+<U000E011E> IGNORE;IGNORE;IGNORE;<U000E011E> % VARIATION SELECTOR-47
+<U000E011F> IGNORE;IGNORE;IGNORE;<U000E011F> % VARIATION SELECTOR-48
+<U000E0120> IGNORE;IGNORE;IGNORE;<U000E0120> % VARIATION SELECTOR-49
+<U000E0121> IGNORE;IGNORE;IGNORE;<U000E0121> % VARIATION SELECTOR-50
+<U000E0122> IGNORE;IGNORE;IGNORE;<U000E0122> % VARIATION SELECTOR-51
+<U000E0123> IGNORE;IGNORE;IGNORE;<U000E0123> % VARIATION SELECTOR-52
+<U000E0124> IGNORE;IGNORE;IGNORE;<U000E0124> % VARIATION SELECTOR-53
+<U000E0125> IGNORE;IGNORE;IGNORE;<U000E0125> % VARIATION SELECTOR-54
+<U000E0126> IGNORE;IGNORE;IGNORE;<U000E0126> % VARIATION SELECTOR-55
+<U000E0127> IGNORE;IGNORE;IGNORE;<U000E0127> % VARIATION SELECTOR-56
+<U000E0128> IGNORE;IGNORE;IGNORE;<U000E0128> % VARIATION SELECTOR-57
+<U000E0129> IGNORE;IGNORE;IGNORE;<U000E0129> % VARIATION SELECTOR-58
+<U000E012A> IGNORE;IGNORE;IGNORE;<U000E012A> % VARIATION SELECTOR-59
+<U000E012B> IGNORE;IGNORE;IGNORE;<U000E012B> % VARIATION SELECTOR-60
+<U000E012C> IGNORE;IGNORE;IGNORE;<U000E012C> % VARIATION SELECTOR-61
+<U000E012D> IGNORE;IGNORE;IGNORE;<U000E012D> % VARIATION SELECTOR-62
+<U000E012E> IGNORE;IGNORE;IGNORE;<U000E012E> % VARIATION SELECTOR-63
+<U000E012F> IGNORE;IGNORE;IGNORE;<U000E012F> % VARIATION SELECTOR-64
+<U000E0130> IGNORE;IGNORE;IGNORE;<U000E0130> % VARIATION SELECTOR-65
+<U000E0131> IGNORE;IGNORE;IGNORE;<U000E0131> % VARIATION SELECTOR-66
+<U000E0132> IGNORE;IGNORE;IGNORE;<U000E0132> % VARIATION SELECTOR-67
+<U000E0133> IGNORE;IGNORE;IGNORE;<U000E0133> % VARIATION SELECTOR-68
+<U000E0134> IGNORE;IGNORE;IGNORE;<U000E0134> % VARIATION SELECTOR-69
+<U000E0135> IGNORE;IGNORE;IGNORE;<U000E0135> % VARIATION SELECTOR-70
+<U000E0136> IGNORE;IGNORE;IGNORE;<U000E0136> % VARIATION SELECTOR-71
+<U000E0137> IGNORE;IGNORE;IGNORE;<U000E0137> % VARIATION SELECTOR-72
+<U000E0138> IGNORE;IGNORE;IGNORE;<U000E0138> % VARIATION SELECTOR-73
+<U000E0139> IGNORE;IGNORE;IGNORE;<U000E0139> % VARIATION SELECTOR-74
+<U000E013A> IGNORE;IGNORE;IGNORE;<U000E013A> % VARIATION SELECTOR-75
+<U000E013B> IGNORE;IGNORE;IGNORE;<U000E013B> % VARIATION SELECTOR-76
+<U000E013C> IGNORE;IGNORE;IGNORE;<U000E013C> % VARIATION SELECTOR-77
+<U000E013D> IGNORE;IGNORE;IGNORE;<U000E013D> % VARIATION SELECTOR-78
+<U000E013E> IGNORE;IGNORE;IGNORE;<U000E013E> % VARIATION SELECTOR-79
+<U000E013F> IGNORE;IGNORE;IGNORE;<U000E013F> % VARIATION SELECTOR-80
+<U000E0140> IGNORE;IGNORE;IGNORE;<U000E0140> % VARIATION SELECTOR-81
+<U000E0141> IGNORE;IGNORE;IGNORE;<U000E0141> % VARIATION SELECTOR-82
+<U000E0142> IGNORE;IGNORE;IGNORE;<U000E0142> % VARIATION SELECTOR-83
+<U000E0143> IGNORE;IGNORE;IGNORE;<U000E0143> % VARIATION SELECTOR-84
+<U000E0144> IGNORE;IGNORE;IGNORE;<U000E0144> % VARIATION SELECTOR-85
+<U000E0145> IGNORE;IGNORE;IGNORE;<U000E0145> % VARIATION SELECTOR-86
+<U000E0146> IGNORE;IGNORE;IGNORE;<U000E0146> % VARIATION SELECTOR-87
+<U000E0147> IGNORE;IGNORE;IGNORE;<U000E0147> % VARIATION SELECTOR-88
+<U000E0148> IGNORE;IGNORE;IGNORE;<U000E0148> % VARIATION SELECTOR-89
+<U000E0149> IGNORE;IGNORE;IGNORE;<U000E0149> % VARIATION SELECTOR-90
+<U000E014A> IGNORE;IGNORE;IGNORE;<U000E014A> % VARIATION SELECTOR-91
+<U000E014B> IGNORE;IGNORE;IGNORE;<U000E014B> % VARIATION SELECTOR-92
+<U000E014C> IGNORE;IGNORE;IGNORE;<U000E014C> % VARIATION SELECTOR-93
+<U000E014D> IGNORE;IGNORE;IGNORE;<U000E014D> % VARIATION SELECTOR-94
+<U000E014E> IGNORE;IGNORE;IGNORE;<U000E014E> % VARIATION SELECTOR-95
+<U000E014F> IGNORE;IGNORE;IGNORE;<U000E014F> % VARIATION SELECTOR-96
+<U000E0150> IGNORE;IGNORE;IGNORE;<U000E0150> % VARIATION SELECTOR-97
+<U000E0151> IGNORE;IGNORE;IGNORE;<U000E0151> % VARIATION SELECTOR-98
+<U000E0152> IGNORE;IGNORE;IGNORE;<U000E0152> % VARIATION SELECTOR-99
+<U000E0153> IGNORE;IGNORE;IGNORE;<U000E0153> % VARIATION SELECTOR-100
+<U000E0154> IGNORE;IGNORE;IGNORE;<U000E0154> % VARIATION SELECTOR-101
+<U000E0155> IGNORE;IGNORE;IGNORE;<U000E0155> % VARIATION SELECTOR-102
+<U000E0156> IGNORE;IGNORE;IGNORE;<U000E0156> % VARIATION SELECTOR-103
+<U000E0157> IGNORE;IGNORE;IGNORE;<U000E0157> % VARIATION SELECTOR-104
+<U000E0158> IGNORE;IGNORE;IGNORE;<U000E0158> % VARIATION SELECTOR-105
+<U000E0159> IGNORE;IGNORE;IGNORE;<U000E0159> % VARIATION SELECTOR-106
+<U000E015A> IGNORE;IGNORE;IGNORE;<U000E015A> % VARIATION SELECTOR-107
+<U000E015B> IGNORE;IGNORE;IGNORE;<U000E015B> % VARIATION SELECTOR-108
+<U000E015C> IGNORE;IGNORE;IGNORE;<U000E015C> % VARIATION SELECTOR-109
+<U000E015D> IGNORE;IGNORE;IGNORE;<U000E015D> % VARIATION SELECTOR-110
+<U000E015E> IGNORE;IGNORE;IGNORE;<U000E015E> % VARIATION SELECTOR-111
+<U000E015F> IGNORE;IGNORE;IGNORE;<U000E015F> % VARIATION SELECTOR-112
+<U000E0160> IGNORE;IGNORE;IGNORE;<U000E0160> % VARIATION SELECTOR-113
+<U000E0161> IGNORE;IGNORE;IGNORE;<U000E0161> % VARIATION SELECTOR-114
+<U000E0162> IGNORE;IGNORE;IGNORE;<U000E0162> % VARIATION SELECTOR-115
+<U000E0163> IGNORE;IGNORE;IGNORE;<U000E0163> % VARIATION SELECTOR-116
+<U000E0164> IGNORE;IGNORE;IGNORE;<U000E0164> % VARIATION SELECTOR-117
+<U000E0165> IGNORE;IGNORE;IGNORE;<U000E0165> % VARIATION SELECTOR-118
+<U000E0166> IGNORE;IGNORE;IGNORE;<U000E0166> % VARIATION SELECTOR-119
+<U000E0167> IGNORE;IGNORE;IGNORE;<U000E0167> % VARIATION SELECTOR-120
+<U000E0168> IGNORE;IGNORE;IGNORE;<U000E0168> % VARIATION SELECTOR-121
+<U000E0169> IGNORE;IGNORE;IGNORE;<U000E0169> % VARIATION SELECTOR-122
+<U000E016A> IGNORE;IGNORE;IGNORE;<U000E016A> % VARIATION SELECTOR-123
+<U000E016B> IGNORE;IGNORE;IGNORE;<U000E016B> % VARIATION SELECTOR-124
+<U000E016C> IGNORE;IGNORE;IGNORE;<U000E016C> % VARIATION SELECTOR-125
+<U000E016D> IGNORE;IGNORE;IGNORE;<U000E016D> % VARIATION SELECTOR-126
+<U000E016E> IGNORE;IGNORE;IGNORE;<U000E016E> % VARIATION SELECTOR-127
+<U000E016F> IGNORE;IGNORE;IGNORE;<U000E016F> % VARIATION SELECTOR-128
+<U000E0170> IGNORE;IGNORE;IGNORE;<U000E0170> % VARIATION SELECTOR-129
+<U000E0171> IGNORE;IGNORE;IGNORE;<U000E0171> % VARIATION SELECTOR-130
+<U000E0172> IGNORE;IGNORE;IGNORE;<U000E0172> % VARIATION SELECTOR-131
+<U000E0173> IGNORE;IGNORE;IGNORE;<U000E0173> % VARIATION SELECTOR-132
+<U000E0174> IGNORE;IGNORE;IGNORE;<U000E0174> % VARIATION SELECTOR-133
+<U000E0175> IGNORE;IGNORE;IGNORE;<U000E0175> % VARIATION SELECTOR-134
+<U000E0176> IGNORE;IGNORE;IGNORE;<U000E0176> % VARIATION SELECTOR-135
+<U000E0177> IGNORE;IGNORE;IGNORE;<U000E0177> % VARIATION SELECTOR-136
+<U000E0178> IGNORE;IGNORE;IGNORE;<U000E0178> % VARIATION SELECTOR-137
+<U000E0179> IGNORE;IGNORE;IGNORE;<U000E0179> % VARIATION SELECTOR-138
+<U000E017A> IGNORE;IGNORE;IGNORE;<U000E017A> % VARIATION SELECTOR-139
+<U000E017B> IGNORE;IGNORE;IGNORE;<U000E017B> % VARIATION SELECTOR-140
+<U000E017C> IGNORE;IGNORE;IGNORE;<U000E017C> % VARIATION SELECTOR-141
+<U000E017D> IGNORE;IGNORE;IGNORE;<U000E017D> % VARIATION SELECTOR-142
+<U000E017E> IGNORE;IGNORE;IGNORE;<U000E017E> % VARIATION SELECTOR-143
+<U000E017F> IGNORE;IGNORE;IGNORE;<U000E017F> % VARIATION SELECTOR-144
+<U000E0180> IGNORE;IGNORE;IGNORE;<U000E0180> % VARIATION SELECTOR-145
+<U000E0181> IGNORE;IGNORE;IGNORE;<U000E0181> % VARIATION SELECTOR-146
+<U000E0182> IGNORE;IGNORE;IGNORE;<U000E0182> % VARIATION SELECTOR-147
+<U000E0183> IGNORE;IGNORE;IGNORE;<U000E0183> % VARIATION SELECTOR-148
+<U000E0184> IGNORE;IGNORE;IGNORE;<U000E0184> % VARIATION SELECTOR-149
+<U000E0185> IGNORE;IGNORE;IGNORE;<U000E0185> % VARIATION SELECTOR-150
+<U000E0186> IGNORE;IGNORE;IGNORE;<U000E0186> % VARIATION SELECTOR-151
+<U000E0187> IGNORE;IGNORE;IGNORE;<U000E0187> % VARIATION SELECTOR-152
+<U000E0188> IGNORE;IGNORE;IGNORE;<U000E0188> % VARIATION SELECTOR-153
+<U000E0189> IGNORE;IGNORE;IGNORE;<U000E0189> % VARIATION SELECTOR-154
+<U000E018A> IGNORE;IGNORE;IGNORE;<U000E018A> % VARIATION SELECTOR-155
+<U000E018B> IGNORE;IGNORE;IGNORE;<U000E018B> % VARIATION SELECTOR-156
+<U000E018C> IGNORE;IGNORE;IGNORE;<U000E018C> % VARIATION SELECTOR-157
+<U000E018D> IGNORE;IGNORE;IGNORE;<U000E018D> % VARIATION SELECTOR-158
+<U000E018E> IGNORE;IGNORE;IGNORE;<U000E018E> % VARIATION SELECTOR-159
+<U000E018F> IGNORE;IGNORE;IGNORE;<U000E018F> % VARIATION SELECTOR-160
+<U000E0190> IGNORE;IGNORE;IGNORE;<U000E0190> % VARIATION SELECTOR-161
+<U000E0191> IGNORE;IGNORE;IGNORE;<U000E0191> % VARIATION SELECTOR-162
+<U000E0192> IGNORE;IGNORE;IGNORE;<U000E0192> % VARIATION SELECTOR-163
+<U000E0193> IGNORE;IGNORE;IGNORE;<U000E0193> % VARIATION SELECTOR-164
+<U000E0194> IGNORE;IGNORE;IGNORE;<U000E0194> % VARIATION SELECTOR-165
+<U000E0195> IGNORE;IGNORE;IGNORE;<U000E0195> % VARIATION SELECTOR-166
+<U000E0196> IGNORE;IGNORE;IGNORE;<U000E0196> % VARIATION SELECTOR-167
+<U000E0197> IGNORE;IGNORE;IGNORE;<U000E0197> % VARIATION SELECTOR-168
+<U000E0198> IGNORE;IGNORE;IGNORE;<U000E0198> % VARIATION SELECTOR-169
+<U000E0199> IGNORE;IGNORE;IGNORE;<U000E0199> % VARIATION SELECTOR-170
+<U000E019A> IGNORE;IGNORE;IGNORE;<U000E019A> % VARIATION SELECTOR-171
+<U000E019B> IGNORE;IGNORE;IGNORE;<U000E019B> % VARIATION SELECTOR-172
+<U000E019C> IGNORE;IGNORE;IGNORE;<U000E019C> % VARIATION SELECTOR-173
+<U000E019D> IGNORE;IGNORE;IGNORE;<U000E019D> % VARIATION SELECTOR-174
+<U000E019E> IGNORE;IGNORE;IGNORE;<U000E019E> % VARIATION SELECTOR-175
+<U000E019F> IGNORE;IGNORE;IGNORE;<U000E019F> % VARIATION SELECTOR-176
+<U000E01A0> IGNORE;IGNORE;IGNORE;<U000E01A0> % VARIATION SELECTOR-177
+<U000E01A1> IGNORE;IGNORE;IGNORE;<U000E01A1> % VARIATION SELECTOR-178
+<U000E01A2> IGNORE;IGNORE;IGNORE;<U000E01A2> % VARIATION SELECTOR-179
+<U000E01A3> IGNORE;IGNORE;IGNORE;<U000E01A3> % VARIATION SELECTOR-180
+<U000E01A4> IGNORE;IGNORE;IGNORE;<U000E01A4> % VARIATION SELECTOR-181
+<U000E01A5> IGNORE;IGNORE;IGNORE;<U000E01A5> % VARIATION SELECTOR-182
+<U000E01A6> IGNORE;IGNORE;IGNORE;<U000E01A6> % VARIATION SELECTOR-183
+<U000E01A7> IGNORE;IGNORE;IGNORE;<U000E01A7> % VARIATION SELECTOR-184
+<U000E01A8> IGNORE;IGNORE;IGNORE;<U000E01A8> % VARIATION SELECTOR-185
+<U000E01A9> IGNORE;IGNORE;IGNORE;<U000E01A9> % VARIATION SELECTOR-186
+<U000E01AA> IGNORE;IGNORE;IGNORE;<U000E01AA> % VARIATION SELECTOR-187
+<U000E01AB> IGNORE;IGNORE;IGNORE;<U000E01AB> % VARIATION SELECTOR-188
+<U000E01AC> IGNORE;IGNORE;IGNORE;<U000E01AC> % VARIATION SELECTOR-189
+<U000E01AD> IGNORE;IGNORE;IGNORE;<U000E01AD> % VARIATION SELECTOR-190
+<U000E01AE> IGNORE;IGNORE;IGNORE;<U000E01AE> % VARIATION SELECTOR-191
+<U000E01AF> IGNORE;IGNORE;IGNORE;<U000E01AF> % VARIATION SELECTOR-192
+<U000E01B0> IGNORE;IGNORE;IGNORE;<U000E01B0> % VARIATION SELECTOR-193
+<U000E01B1> IGNORE;IGNORE;IGNORE;<U000E01B1> % VARIATION SELECTOR-194
+<U000E01B2> IGNORE;IGNORE;IGNORE;<U000E01B2> % VARIATION SELECTOR-195
+<U000E01B3> IGNORE;IGNORE;IGNORE;<U000E01B3> % VARIATION SELECTOR-196
+<U000E01B4> IGNORE;IGNORE;IGNORE;<U000E01B4> % VARIATION SELECTOR-197
+<U000E01B5> IGNORE;IGNORE;IGNORE;<U000E01B5> % VARIATION SELECTOR-198
+<U000E01B6> IGNORE;IGNORE;IGNORE;<U000E01B6> % VARIATION SELECTOR-199
+<U000E01B7> IGNORE;IGNORE;IGNORE;<U000E01B7> % VARIATION SELECTOR-200
+<U000E01B8> IGNORE;IGNORE;IGNORE;<U000E01B8> % VARIATION SELECTOR-201
+<U000E01B9> IGNORE;IGNORE;IGNORE;<U000E01B9> % VARIATION SELECTOR-202
+<U000E01BA> IGNORE;IGNORE;IGNORE;<U000E01BA> % VARIATION SELECTOR-203
+<U000E01BB> IGNORE;IGNORE;IGNORE;<U000E01BB> % VARIATION SELECTOR-204
+<U000E01BC> IGNORE;IGNORE;IGNORE;<U000E01BC> % VARIATION SELECTOR-205
+<U000E01BD> IGNORE;IGNORE;IGNORE;<U000E01BD> % VARIATION SELECTOR-206
+<U000E01BE> IGNORE;IGNORE;IGNORE;<U000E01BE> % VARIATION SELECTOR-207
+<U000E01BF> IGNORE;IGNORE;IGNORE;<U000E01BF> % VARIATION SELECTOR-208
+<U000E01C0> IGNORE;IGNORE;IGNORE;<U000E01C0> % VARIATION SELECTOR-209
+<U000E01C1> IGNORE;IGNORE;IGNORE;<U000E01C1> % VARIATION SELECTOR-210
+<U000E01C2> IGNORE;IGNORE;IGNORE;<U000E01C2> % VARIATION SELECTOR-211
+<U000E01C3> IGNORE;IGNORE;IGNORE;<U000E01C3> % VARIATION SELECTOR-212
+<U000E01C4> IGNORE;IGNORE;IGNORE;<U000E01C4> % VARIATION SELECTOR-213
+<U000E01C5> IGNORE;IGNORE;IGNORE;<U000E01C5> % VARIATION SELECTOR-214
+<U000E01C6> IGNORE;IGNORE;IGNORE;<U000E01C6> % VARIATION SELECTOR-215
+<U000E01C7> IGNORE;IGNORE;IGNORE;<U000E01C7> % VARIATION SELECTOR-216
+<U000E01C8> IGNORE;IGNORE;IGNORE;<U000E01C8> % VARIATION SELECTOR-217
+<U000E01C9> IGNORE;IGNORE;IGNORE;<U000E01C9> % VARIATION SELECTOR-218
+<U000E01CA> IGNORE;IGNORE;IGNORE;<U000E01CA> % VARIATION SELECTOR-219
+<U000E01CB> IGNORE;IGNORE;IGNORE;<U000E01CB> % VARIATION SELECTOR-220
+<U000E01CC> IGNORE;IGNORE;IGNORE;<U000E01CC> % VARIATION SELECTOR-221
+<U000E01CD> IGNORE;IGNORE;IGNORE;<U000E01CD> % VARIATION SELECTOR-222
+<U000E01CE> IGNORE;IGNORE;IGNORE;<U000E01CE> % VARIATION SELECTOR-223
+<U000E01CF> IGNORE;IGNORE;IGNORE;<U000E01CF> % VARIATION SELECTOR-224
+<U000E01D0> IGNORE;IGNORE;IGNORE;<U000E01D0> % VARIATION SELECTOR-225
+<U000E01D1> IGNORE;IGNORE;IGNORE;<U000E01D1> % VARIATION SELECTOR-226
+<U000E01D2> IGNORE;IGNORE;IGNORE;<U000E01D2> % VARIATION SELECTOR-227
+<U000E01D3> IGNORE;IGNORE;IGNORE;<U000E01D3> % VARIATION SELECTOR-228
+<U000E01D4> IGNORE;IGNORE;IGNORE;<U000E01D4> % VARIATION SELECTOR-229
+<U000E01D5> IGNORE;IGNORE;IGNORE;<U000E01D5> % VARIATION SELECTOR-230
+<U000E01D6> IGNORE;IGNORE;IGNORE;<U000E01D6> % VARIATION SELECTOR-231
+<U000E01D7> IGNORE;IGNORE;IGNORE;<U000E01D7> % VARIATION SELECTOR-232
+<U000E01D8> IGNORE;IGNORE;IGNORE;<U000E01D8> % VARIATION SELECTOR-233
+<U000E01D9> IGNORE;IGNORE;IGNORE;<U000E01D9> % VARIATION SELECTOR-234
+<U000E01DA> IGNORE;IGNORE;IGNORE;<U000E01DA> % VARIATION SELECTOR-235
+<U000E01DB> IGNORE;IGNORE;IGNORE;<U000E01DB> % VARIATION SELECTOR-236
+<U000E01DC> IGNORE;IGNORE;IGNORE;<U000E01DC> % VARIATION SELECTOR-237
+<U000E01DD> IGNORE;IGNORE;IGNORE;<U000E01DD> % VARIATION SELECTOR-238
+<U000E01DE> IGNORE;IGNORE;IGNORE;<U000E01DE> % VARIATION SELECTOR-239
+<U000E01DF> IGNORE;IGNORE;IGNORE;<U000E01DF> % VARIATION SELECTOR-240
+<U000E01E0> IGNORE;IGNORE;IGNORE;<U000E01E0> % VARIATION SELECTOR-241
+<U000E01E1> IGNORE;IGNORE;IGNORE;<U000E01E1> % VARIATION SELECTOR-242
+<U000E01E2> IGNORE;IGNORE;IGNORE;<U000E01E2> % VARIATION SELECTOR-243
+<U000E01E3> IGNORE;IGNORE;IGNORE;<U000E01E3> % VARIATION SELECTOR-244
+<U000E01E4> IGNORE;IGNORE;IGNORE;<U000E01E4> % VARIATION SELECTOR-245
+<U000E01E5> IGNORE;IGNORE;IGNORE;<U000E01E5> % VARIATION SELECTOR-246
+<U000E01E6> IGNORE;IGNORE;IGNORE;<U000E01E6> % VARIATION SELECTOR-247
+<U000E01E7> IGNORE;IGNORE;IGNORE;<U000E01E7> % VARIATION SELECTOR-248
+<U000E01E8> IGNORE;IGNORE;IGNORE;<U000E01E8> % VARIATION SELECTOR-249
+<U000E01E9> IGNORE;IGNORE;IGNORE;<U000E01E9> % VARIATION SELECTOR-250
+<U000E01EA> IGNORE;IGNORE;IGNORE;<U000E01EA> % VARIATION SELECTOR-251
+<U000E01EB> IGNORE;IGNORE;IGNORE;<U000E01EB> % VARIATION SELECTOR-252
+<U000E01EC> IGNORE;IGNORE;IGNORE;<U000E01EC> % VARIATION SELECTOR-253
+<U000E01ED> IGNORE;IGNORE;IGNORE;<U000E01ED> % VARIATION SELECTOR-254
+<U000E01EE> IGNORE;IGNORE;IGNORE;<U000E01EE> % VARIATION SELECTOR-255
+<U000E01EF> IGNORE;IGNORE;IGNORE;<U000E01EF> % VARIATION SELECTOR-256
 <U0009> IGNORE;IGNORE;IGNORE;<U0009> % HORIZONTAL TABULATION (in ISO 6429)
 <U000A> IGNORE;IGNORE;IGNORE;<U000A> % LINE FEED (in ISO 6429)
 <U000B> IGNORE;IGNORE;IGNORE;<U000B> % VERTICAL TABULATION (in ISO 6429)
